### PR TITLE
AVS 5.1: participants change

### PR DESCRIPTION
**Pull request by vytis**
## What's new in this PR?

A slightly improved API was added to handle participants in the call. It directly provides a list of participants in a JSON instead of relying on C structs.

### CODE REVIEW

File: Source/Calling/WireCallCenterV3+Events.swift [hunk 1]
```diff
@@ -170,12 +170,31 @@ extension WireCallCenterV3 {
             self.isReady = true
         }
     }
-
-    /// Handles other users joining / leaving / connecting.
-    func handleGroupMemberChange(conversationId: UUID) {
-        handleEvent("group-member-change") {
-            let members = self.avsWrapper.members(in: conversationId)
```
Comment (typfel) [inline Source/Calling/WireCallCenterV3+Events.swift]: Can the `func members(in conversationId: UUID)` and `AVSCallMember` be deleted now?

Comment (vytis) [reply Source/Calling/WireCallCenterV3+Events.swift]: Good point, will check

Comment (vytis) [reply Source/Calling/WireCallCenterV3+Events.swift]: `AVSCallMember` call member is used in quite a few places, but removed the function.

Comment (typfel) [reply Source/Calling/WireCallCenterV3+Events.swift]: Ah right, so the new `handleParticipantChange` still creates AVSCallMembers.

**Pull request merged**